### DISCUSSION TITLE
When logging in to the same account in two places at the same time, user will automatically log out after a while sometimes.

### DISCUSSION THREAD

**Discussion by joe960913** [supabase/supabase#31279] [Questions]
When the user logs in to the same account on the web client and extension at the same time, both sides will sometimes log out automatically. Is there any best practice for similar situations?

**Comment by GaryAustin1** [comment]
Maybe be clearer on what you are doing with two logins to two places at same time.   Normally a login will generate a session and return an access token and refresh token.  That pair can only be used in one Supabase client.  If you somehow use the access and refresh token in another client then eventually both will get logged out as the refresh token can only be used once.

If you truly sign in with auth sign in methods on two different clients and the do not share local storage or a cookie then they will each have their own pair of tokens and not conflict with each other.
